include/sys/socket.h

```diff
@@ -279,6 +279,12 @@
 #define SCM_CREDENTIALS 0x02    /* rw: struct ucred */
 #define SCM_SECURITY    0x03    /* rw: security label */
 
+/* Desired design of maximum size and alignment (see RFC2553) */
+
+#define SS_MAXSIZE      128  /* Implementation specific max size */
+#define SS_ALIGNSIZE    (sizeof(FAR struct sockaddr *))
+                             /* Implementation specific desired alignment */
+
```

Comment:
Should be interesting to add a DEBUG_ASSERTION to check if SS_MAXSIZE is greater than sa_family_t !

Reply:
Don't need the compiler will stop and complain that the array length is negative.